#key error
dic={"one":"1","two":"2","three":"3"}
print(dic["five"])
#file not found
fp=open("sathya,txt")